packages/react-components/src/components/styles/InputBoxComponent.style.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { mergeStyles, IStyle } from '@fluentui/react';
+
+export const inputBoxWrapperStyle = mergeStyles({
+  padding: '0'
+});
+
+export const inputBoxStyle = mergeStyles({
+  minHeight: '2.25rem', // prevents the input text box from being sized to 0px when the meeting composite chat pane is closed.
+  maxHeight: '8.25rem',
+  outline: 'red 5px',
+  fontWeight: 400,
+  fontSize: '0.875rem',
```

Comment:
I would suggest using the theme.font property for using standardized font weight and size here.

Reply:
For weight - sure!

For size - are we allowing our customers to override font size - it could entirely mess the composite layout up if they set it too big/small